client/ayon_core/modules/deadline/plugins/publish/submit_blender_deadline.py

```diff
@@ -40,6 +40,17 @@ class BlenderSubmitDeadline(abstract_submit_deadline.AbstractSubmitDeadline,
     group = None
     job_delay = "00:00:00:00"
 
+    @classmethod
+    def apply_settings(cls, project_settings):
+        settings = project_settings["deadline"]["publish"]["BlenderSubmitDeadline"]  # noqa
+
+        # Take some defaults from settings
+        cls.asset_dependencies = settings.get("asset_dependencies",
+                                              cls.asset_dependencies)
+        cls.use_published = settings.get("use_published", cls.use_published)
+        cls.priority = settings.get("priority", cls.priority)
+        cls.group = settings.get("group", cls.group)
```

Comment:
This should be redundant and be the default behavior if `apply_settings` is not defined. Right @iLLiCiTiT ?

Reply:
Also, does this mean that `chunk_size` does not come from settings since it's not defined here? :)

Reply:
it should be working as expected as there is similar to use `apply_settings` in `ExtractReviewIntermediates` plugin

Reply:
I'm sure the `apply_settings` works, but by design if you do not implement the method it defaults automatically to what it seems to be doing here currently. As such, I believe it should be completely redundant.

Reply:
Well, from my pov it was more aligning Blender to Maya, not really a silent fix 😅 but I got the message, waiting for some answer from @iLLiCiTiT :)

Reply:
Long story short you should not need to override the method.

Reply:
This comment still applies I believe.